### ISSUE TITLE
Accessibility issue - aria-hidden-focus 

### ISSUE THREAD

**Issue by spacedmonkey** [ampproject/amp-wp#3305]
When running the automated accessibility tests in #3294 the follow aria-hidden-focus  errors where found with story editor. 
```
 Rule: "aria-hidden-focus" (ARIA hidden element must not contain focusable elements)
    Affected Nodes:
      #block-380cff1c-f06a-4c44-a949-14e29c379349 > .editor-block-list__block-edit.block-editor-block-list__block-edit > div > div > div > .amp-story-editor__rotate-container > .block-mover > div > div[draggable="true"][aria-hidden="true"]
        Fix ALL of the following:
          - Focusable content should be disabled or be removed from the DOM.
      .components-resizable-box__container.amp-story-resize-container[height="\36 0"] > .amp-story-editor__rotate-container > .block-mover > div > div[draggable="true"][aria-hidden="true"]
        Fix ALL of the following:
          - Focusable content should be disabled or be removed from the DOM.
      div[height="\32 00"] > .amp-story-editor__rotate-container > .block-mover > div > div[draggable="true"][aria-hidden="true"]
        Fix ALL of the following:
          - Focusable content should be disabled or be removed from the DOM.
```
```
  Rule: "aria-hidden-focus" (ARIA hidden element must not contain focusable elements)
    Affected Nodes:
      .amp-story-editor__rotate-container > .block-mover > div > div[draggable="true"][aria-hidden="true"]
        Fix ALL of the following:
          - Focusable content should be disabled or be removed from the DOM.
```

Help: https://dequeuniversity.com/rules/axe/3.3/aria-hidden-focus?application=axe-puppeteer

**Comment by jauyong** [comment]
Hey @jasti , I wasn't sure what AC to put for this ticket. 
Would something like "address the aria-hidden-focus errors found by the automated accessibility test" suffice? Or I could list out the things it says to fix as AC?
